app-emulation/wine-proton/files/README.gentoo

```diff
@@ -0,0 +1,52 @@
+While this version of Wine is usable independently, it does contain some
+Steam-oriented changes that may be unexpected. Notably C:\users\steamuser
+is used rather than C:\users\<user-name> and it may be necessary to move
+files if converting from non-proton Wine prefixes (like games' save files).
+
+To keep a low maintenance cost, these changes are not reverted in Gentoo
+with the exception of re-enabling winemenubuilder for desktop entries.
+
+Also, given this Wine variant is oriented toward running games, some
+normally unrelated features (e.g. ldap) may not have real upstream
+(Valve) support. Because of this, some are left always disabled.
+
+If needed, use other variants for the more expected Wine behavior.
+
+---
+
+Given Proton is not creating the Wine prefixes, some features are not
+enabled by default and PROTON_* variables to control them are unrecognized.
+
+To (optionally) setup some notable defaults of Proton:
+- export WINEFSYNC=1
```

Comment:
I think one of the biggest values of this package is to have an environment similar to Steam-created wineprefixes but outside Steam, so if Steam-created wineprefixes enable these by default, I think we should try to enable them by default. 

Can we do quick source patches to change the default value? We should ofc keep it possible to disable them thru envvars tho

Reply:
Hm, I don't know. These variables are fairly well known/documented (even non-proton) as additional patches. And other people/distros packaging wine-proton standalone don't force anything on afaik (at most they revert a steam-specific stuff instead, albeit only one I reverted is winemenubuilder).

Would still need to setup dxvk/vkd3d-proton manually either way, as making that default would make reverting difficult.

Thought a README to just inform of all 4 was fair enough.

Reply:
sorry i only read the first one and assumed they all are just envvars and don't require setup, yeah probably the readme is fine